packages/db/src/project/test/integration.spec.ts

```diff
@@ -19,7 +19,7 @@ import type { Query, Mutation } from "@truffle/db/process";
 
 let server;
 const port = 8545;
-
+// @ts-ignore
```

Comment:
Yup, It's a jest thing because `done` is being supplied to ganache and I thought it overly pedantic to squabble with jest over `any` and also considered it a reasonable externality as we still rely on 20ish other uses of `ts-ignore` in the tests and ~100 or so in the source itself. 

![image](https://user-images.githubusercontent.com/6285466/156822103-bf6b1cd7-89d0-4b34-bd0c-fa8bf689b4d8.png)

Reply:
OK, might be worth expanding the code comment to make this clear.